addons/account/views/account_portal_templates.xml

```diff
@@ -113,10 +113,24 @@
                 <t t-call="portal.portal_record_sidebar">
                     <t t-set="classes" t-value="'col-lg-3 col-xl-4 d-print-none me-lg-auto'"/>
                     <t t-set="title">
-                        <h2>
+                        <h2 t-if="0">
```

Comment:
Sorry I don't understand why you add this condition on the template ? Why not just removing it ? Is it for other templates that might use xpath on it ? Also, why hidding the total amount of the invoice once the invoice is paid ?

Reply:
> why hidding the total amount of the invoice once the invoice is paid
corrected it